app/components/dish.js

```diff
@@ -61,7 +61,7 @@ const styles = StyleSheet.create({
   cardContainer: {
     alignItems: "flex-end",
     alignContent: "stretch",
-    maxHeight: "40%"
+    maxHeight: "100%"
```

Comment:
Could be some conflicts with changing the height since dish cards are used in different places

Reply:
oh shoot i didn't even realize i changed that, i was just seeing what those did lol, i'll revert it

Reply:
So I'm not sure why, but when I reverted the change in dish.js I ended up with this:

![IMG_FA0637C02C2C-1](https://user-images.githubusercontent.com/10168472/106791825-4ba8ec80-660a-11eb-88fe-66a9fcec6d8e.jpeg)

Not sure if that's just on my end or if there's something going on?

Reply:
Is there a chance we could make it horizontal like the dish cards? The dish cards are width: 100% and height: 225

Reply:
for the recommendation card? so like each recommendation takes up the whole screen